command/resource/client/grpc-flags.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package client
```

Comment:
This client is pretty bare bones right now, I am guessing you will add connection pool management in the TLS PR?

Reply:
Correct, this file of `grpc-flags.go` hosts all of the flags related values.